components/raftstore/src/store/async_io/write.rs

```diff
@@ -511,6 +520,12 @@ where
         }
     }
 
+    fn on_after_writer_to_kv_db(&mut self) {
```

Comment:
In v2, all states are written to raft db, so the hook should be called after finishing all writes.

Reply:
yes, I wiil call it after write raft db